README.md

```diff
@@ -47,16 +47,26 @@ docker run -p 6379:6379 -p 3000:3000 -it --rm  -v ./data:/data falkordb/falkordb
 Currently, this SDK supports the following LLMs API:
 
 * [OpenAI](https://openai.com/index/openai-api) Recommended model:`gpt-4o`
-* [google](https://makersuite.google.com/app/apikey) Recommended model:`gemini-1.5-flash-001`
-
-Make sure that a `.env` file is present with all required credentials.
+* [Google](https://makersuite.google.com/app/apikey) Recommended model:`gemini-1.5-flash-001`
+* [Azure-OpenAI](https://ai.azure.com) Recommended model:`gpt-4o`
+* [Ollama](https://ollama.com/) Available only to the Q&A step. Recommended models: `llama3`. Ollama models are suitable for cases where the knowledge graph (KG) is predefined.
+  
+    To get started with Ollama:
+    1. Install Ollama from https://ollama.com/
+    2. Pull your preferred model: `ollama pull llama3`
 
    <details>
      <summary>.env</summary>
   
    ```
    OPENAI_API_KEY="OPENAI_API_KEY"
    GOOGLE_API_KEY="GOOGLE_API_KEY"
+   # API Key from Azure Portal
+   AZURE_OPENAI_API_KEY="AZURE_OPENAI_API_KEY"
+   # Endpoint format: https://{your-resource-name}.openai.azure.com
+   AZURE_ENDPOINT="AZURE_ENDPOINT"
+   # API Version (e.g., 2023-05-15)
+   AZURE_API_VERSION="AZURE_API_VERSION"
    ```
   
    </details>

```

Comment:
Replace this with a `.env.template` file, this is where users are expecting to find this information.